core/cas-server-core-authentication-mfa-api/src/main/java/org/apereo/cas/authentication/MultifactorAuthenticationUtils.java

```diff
@@ -154,10 +156,10 @@ public static MultifactorAuthenticationProviderBypass newMultifactorAuthenticati
         val bypass = new ChainingMultifactorAuthenticationBypassProvider();
         bypass.addBypass(new DefaultMultifactorAuthenticationProviderBypass(props));
 
-        if (props.getType() == MultifactorAuthenticationProviderBypassProperties.MultifactorProviderBypassTypes.GROOVY) {
```

Comment:
Can't make this change in a patch release.

Reply:
Why?  It does not add to or change how properties are set, but allows both Groovy and Rest bypass evaluators be added to the Chain if they are defined.

Reply:
If you are effectively removing the type property, that would mean the property would be removed from the model, which means it will cause errors for folks who have configured that already.